src/Console/Commands/Tenants/AssociateAdminCommand.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using McMaster.Extensions.CommandLineUtils;
+using Microsoft.Extensions.Options;
+using Omnia.CLI.Extensions;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using System.Text;
+using Microsoft.AspNetCore.JsonPatch;
+
+namespace Omnia.CLI.Commands.Tenants
+{
+    [Command(Name = "associate-admin", Description = "Associate admin user to Tenant.")]
```

Comment:
I think we should create a new Command to add a single user to a Role instead of doing this.
In that case, the user would do something like: `omnia-cli security users add -subscription my -tenant mytenant -username myemail@mail.com -role Administration`

Reply:
When the given role is not found, the error message shouldn't be "InternalError: The server encountered an internal error. Please retry the request."